### DISCUSSION TITLE
Query results in Ace differ from results in app

### DISCUSSION THREAD

**Discussion by jordanade** [Sequel-Ace/Sequel-Ace#1702] [Q&A]
I've noticed that sometimes results in Sequel Ace differ from those in my Laravel 9 (PHP 8.1) app, or I'll get an error in Ace that I don't get in my app. I assume this is due to settings. Any tips for how to adjust the settings to match Laravel's?

**Comment by Jason-Morcos** [comment]
How do the results differ exactly? And can you share your Laravel connection settings from the Laravel databases config file (anonymized as needed, and filling envs with real values as possible)?

**Reply by Jason-Morcos** [reply]
And what kind of error have you gotten that you don't see in your app? Is it possible you have strict mode turned off in your Laravel database config? I use Laravel regularly and haven't experienced the type of behavior you're describing.
